_scheduled/README.md

```diff
@@ -0,0 +1,21 @@
+# Scheduled Posts
+In this folder you can save all of your "Completed" blog posts which are scheduled to be uploaded on a specific date.
+> Incomplete posts should be saved in ```_drafts```
+
+### Name Format
+In this folder you need to store your file in the same format as you would in ```_posts```
+
+example file: ```2024-08-26-This file will be uploaded on 26 August.md```
+
+## Important Notes:
+- The scheduler uploads posts every 🕛 00:00 UTC
+- It will only upload posts that are scheduled on their respective days
+- It will only upload files which follow the pattern ```yyyy-mm-dd-title.md```
+  - This means that only markdown files will be posted
+  - It means that any markdown which do not follow this pattern (such as this README.md file) will not be posted
+- The scheduler will only post files to the ```_posts``` folder, it will not post to folders like ```_projects``` or ```_news```
```

Comment:
I would change this to:

- The scheduler works by moving posts from the `_scheduled/` directory to `_posts/`, it will not work for other collections like `_projects/` or `_news/`.

Reply:
Ok